fem/bilinearform.hpp

```diff
@@ -325,6 +325,12 @@ class BilinearForm : public Matrix
    /// Get the finite element space restriction matrix
    virtual const Operator *GetRestriction() const
    { return fes->GetConformingRestriction(); }
+   /// Get the output finite element space prolongation matrix
+   virtual const Operator *GetOutputProlongation() const
+   { return GetProlongation(); }
+   /// Get the output finite element space restriction matrix
+   virtual const Operator *GetOutputRestriction() const
```

Comment:
Should change these from "Output" to "Test" as well or eliminate them.

Reply:
Removed.